resources/index-dev-vscode.html

```diff
@@ -9,7 +9,7 @@
 		<link rel="icon" media="(prefers-color-scheme:dark)" href="/favicon-dark.svg" type="image/svg+xml" />
 		<link rel="manifest" href="/manifest.json" />
 		<title>GitHub1s</title>
-		<script>if (window.location.hostname.match(/\.?gitlab1s\.com$/i)) { window.document.title = 'GitLab1s'; } else if (window.location.hostname.match(/\.?bitbucket1s\.org$/i)) { window.document.title = 'Bitbucket1s'; };</script>
+		<script>if (window.location.hostname.match(/\.?gitlab1s\.com$/i)) { window.document.title = 'GitLab1s'; } else if (window.location.hostname.match(/\.?bitbucket1s\.org$/i)) { window.document.title = 'Bitbucket1s'; } else if (window.location.hostname.match(/\.?npmjs1s\.com$/i)) { window.document.title = 'npm1s'; };</script>
```

Comment:
I think we can refactor this to make it reusable in all `.html` files without copy paste, I will look into it.

Reply:
yes! thank you for your advice,  i think we can migrate these to the build scripts

Reply:
I'm working on this, will share a PR soon.